packages/crypto/src/slip10.ts

```diff
@@ -26,6 +24,12 @@ function bytesToUnsignedBigInt(a: Uint8Array): bigint {
   return BigInt("0x" + toHex(a));
 }
 
+function intTo32be(n: bigint): Uint8Array {
+  // 32 bytes is 64 hexadecimal characters
+  const hex = n.toString(16).padStart(64, "0");
+  return fromHex(hex);
+}
```

Comment:
```suggestion
function uintTo32be(n: bigint): Uint8Array {
  assert(n >= 0n);
  assert(n < BigInt(2**32));
  // 32 bytes is 64 hexadecimal characters
  const hex = n.toString(16).padStart(64, "0");
  return fromHex(hex);
}
```

The implementation is not correct for values outside of the uint32 range. So let's just assert that. (we need to import asser from @cosmjs/utils)

Reply:
Oh, yeah sure you are right